dec.js

```diff
@@ -0,0 +1,29 @@
+import isNumber from './isNumber.js'
+
+/**
+ * Decrease the value of a `number`.
+ *
+ * @since 4.17.15
+ * @category Math
+ * @param {number} number The number to decrease.
+ * @param {number} number Decrease.
+ * @returns {number} Returns the decrease.
+ * @example
+ *
+ * dec(4)
+ * // => 5
```

Comment:
I randomly stumbled across this here 😄 I think you meant 3? 🙂 
```suggestion
 * // => 3
```